cmd/osmosisd/cmd/init.go

```diff
@@ -80,7 +80,10 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
 
+			config.P2P.Seeds = "085f62d67bbf9c501e8ac84d4533440a1eef6c45@95.217.196.54:26656"
+			config.P2P.PersistentPeers = "2e3e3b7703a598024a2fb287587095bc4d14fe52@95.217.196.54:2000,f5be19f84deb843c18e9b612b7987138ba13ac02@5.9.106.185:2000,f9c49739f0641a0a673e7a1e8edc38054fefc840@144.76.183.180:2000,40aafcd9b6959d58dd1c567d9daf2a82a23311cf@162.55.132.230:2000,3fea02d121cb24503d5fbc53216a527257a9ab55@143.198.145.208:26656,e7916387e05acd53d1b8c0f842c13def365c7bb6@176.9.64.212:26656,785bc83577e3980545bac051de8f57a9fd82695f@194.233.164.146:26656,421e7ffd6c239cc51620e2ec2330d392a7baeeb3@207.246.109.0:26656,778fdedf6effe996f039f22901a3360bc838b52e@161.97.187.189:36656,2115945f074ddb038de5d835e287fa03e32f0628@95.217.43.85:26656,a79b9904461a98cdb98a5c15d7d75b02a55b6ab9@162.216.4.214:36656,1639d885f47c96c5cc4be6e4443ba53cdc74fc71@23.111.129.54:36656"
```

Comment:
peers and seed can change over time.
I think it's better to keep this blank and use external documentation.

Reply:
Sir, that's the point.  They will change over time.  

They will change here, and be versioned, and there will be some sense of accountability and we won't have to pass around a semi-secret, hard to locate document.

Reply:
please consider the harm done by asking 100+ people to routinely configure seeds and peers.  

This also allows us to begin working on formally tracking sync speed:

https://github.com/osmosis-labs/osmosis/pull/418

....and shouldn't we live in a world, where each validator has at least one commit on this repository?

Reply:
it's not something who need to be versioned.
it's a production context.

Reply:
2 things:

https://twitter.com/gadikian/status/1427959398857543687

<img width="297" alt="Screen Shot 2021-08-18 at 6 51 55 PM" src="https://user-images.githubusercontent.com/7142025/129893582-3a333e73-7936-4e14-8486-ea591a0af3fd.png">

^ thought we had a team just shy of 100?